package-setup.js

```diff
@@ -0,0 +1,330 @@
+// Copyright 2018 Google LLC
```

Comment:
@crwilcox @alexander-fenster is this file supposed to be here?

Reply:
No, this file is our internal stuff and should not be committed. Please remove it.